omero/sysadmins/version-requirements.rst

```diff
@@ -70,18 +74,27 @@ Operating systems
        - OMERO 5.4
        - OMERO 5.5
        - OMERO 5.6
+       - OMERO 6.0
      * - 14.04
        - |Supported|
        - |Supported|
        - |Dropped|
+       - |Dropped|
      * - 16.04
        - |Recommended|
        - |Recommended|
        - |Supported|
```

Comment:
As discussed during today's call, for the target of the December beta release, we might want to call this `|Unsupported|` and review the level of support either for 5.6.0 GA or in the 5.6.x series. /cc @jburel @manics